pom.xml

```diff
@@ -60,6 +60,12 @@
             <version>4.10</version>
             <scope>test</scope>
         </dependency>
+        <dependency> 
+            <groupId>commons-codec</groupId> 
+            <artifactId>commons-codec</artifactId> 
+            <version>1.10</version> 
+            <scope>test</scope>
+         </dependency>
```

Comment:
This line has a hidden `<U+2028>` before `</dependency>` which throws off the formatting in a plain text editor like `vim` (my main development editor).  That character also causes an error in maven parsing `pom.xml`.